ecs_pod.go

```diff
@@ -105,22 +105,22 @@ func (s *PodSecret) SetOwned(owned bool) *PodSecret {
 type ECSPodStatus string
 
 const (
-	// Starting indicates that the ECS pod is being prepared to run.
-	Starting ECSPodStatus = "starting"
-	// Running indicates that the ECS pod is actively running.
-	Running ECSPodStatus = "running"
-	// Stopped indicates the that ECS pod is stopped, but all of its resources
-	// are still available.
-	Stopped ECSPodStatus = "stopped"
-	// Deleted indicates that the ECS pod has been cleaned up completely,
+	// StartingStatus indicates that the ECS pod is being prepared to run.
```

Comment:
Maybe it just comes down to personal preference, but I think having `Status` as the prefix rather than the postfix is more clear.

Reply:
This is mostly to make it consistent with the other enums. I don't have a strong preference, but I feel like having the category/type of enum as the suffix makes it more human-readable.

Reply:
Nope, that is fine!